client/me/account/main.jsx

```diff
@@ -110,15 +112,14 @@ const Account = localize(
 		},
 
 		updateColorScheme( colorScheme ) {
-			const settingName = 'calypso_preferences.colorScheme';
-
 			// Set a fallback color scheme if no default value is provided by the API.
 			// This is a workaround that allows us to use userSettings.updateSetting() without an
 			// existing value. Without this workaround the save button wouldn't become active.
 			// TODO: the API should provide a default value, which would make this line obsolete
-			update( this.props.userSettings.settings, settingName, value => value || 'default' );
+			update( this.props.userSettings.settings, colorSchemeKey, value => value || 'default' );
 
-			this.updateUserSetting( settingName, colorScheme );
+			this.props.recordTracksEvent( 'calypso_color_schemes_select', { colorScheme: colorScheme } );
```

Comment:
Tracking each different selection as event `calypso_color_schemes_select` with the color scheme value here.